src/client/stylesheets/style.less

```diff
@@ -325,11 +325,26 @@ hr.wide {
 		border: none;
 		border-radius: 0;
 	  }
-  
+
 	  > a:hover,
 	  > a:focus {
 		text-decoration: none;
 		background-color:@dropdown-link-hover-bg;
 	  }
 	}
-}
\ No newline at end of file
+}
+
+@media only screen and (max-width: 600px) {
+	#searchpage-button-group {
+		display: flex;
```

Comment:
It turns out that, considering the media query, you don't even need flex for what you are doing here.

You can remove this css block if you also remove, in the js file, the `justified` property on the `ButtonGroup` as well as `lg={6} lgOffset={3}` on its parent, and finally find and remove `style={{lineHeight: '1em', whiteSpace: 'normal'}` for the Edition Group button.

That way, I think we achieve the cleanest fix.